test/plugins/plugins.test.js

```diff
@@ -120,137 +120,6 @@ describe('all other plugins', function() {
         });
     });
 
-    describe('context', function() {
```

Comment:
We were spreading plugin tests across multiple files - just moved these into the existing context plugin test file.